addons/a11y/src/components/VisionSimulator.tsx

```diff
@@ -66,6 +71,20 @@ export interface Link {
   onClick: () => void;
 }
 
+const Column = styled.span({
+  display: 'flex',
+  flexDirection: 'column',
+});
+
+const Title = styled.span({
+  textTransform: 'capitalize',
+});
+
+const Description = styled.span({
+  fontSize: '0.9em',
+  color: 'gray',
```

Comment:
Can you change the `fontSize` to `11px` and the `color` to use our theme color `dark`. I think it will be something like: `theme.textMutedColor`? This looks great.